demo/Demo Instructions.md

```diff
@@ -0,0 +1,21 @@
+# Pelorus Demo Instructions
+
+Fork the following repository:
+  - [RedHat COP Container Pipeline](https://github.com/redhat-cop/container-pipelines)
+
+Clone the forked repo using ssh. The demo scrupt will assume the project exists in the directory */home/<user>/projects*. If this is not the case, the demo script can be modified to point to the path where the container-pipelines repo is.
+	
+Make the following changes in container-pipelines/basic-nginx:
```

Comment:
Let's just open a PR in container-pipelines with these changes made so that the user doesn't have to make them every time.

Reply:
Is committing code and redeploying pivotal to the demo? Or would it be better to find another demo that has source code in a repo that isn't also the pipeline code?

Reply:
I think that either way we want to make this change in the upstream pipeline. And yes, it might be easier to understand whats happening if our demo app/pipeline were more self-contained, but we don't have one of those at the moment.

Reply:
Is this sufficient for a PR description?
#### What does this PR do?
This PR removed the promotion input gate and verify deployment to the various stages in basic-nginx. The reasons being for a full autonomous demo, the input gates prevent this and the verify stages get stuck for hours when first deployed.

#### How should this be tested?
Existing testing should be sufficient, but confirmation that the user does not need to confirm promotion and verify there are no verify stages.

#### Is there a relevant Issue open for this?
No

#### Who would you like to review this?
cc: @redhat-cop/containers-approvers